addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/JobUtils.java

```diff
@@ -246,7 +249,7 @@ public static void applyParamsToObject(
                 continue;
             }
 
-            String paramMethodName = "set" + key.toUpperCase().charAt(0) + key.substring(1);
+            String paramMethodName = "set" + key.toUpperCase().charAt(0) + key.substring(1);/** Mapping the setters of all the corresponding parameters of job*/
```

Comment:
I think if these are really necessary they should probably be above the line they apply to. (Throughout)